### PR TITLE
Add GPL 3.0 LICENSE.TXT file and license command

### LINKED ISSUE #19: Add a code license
MIT? GPL?

### PR DESCRIPTION
**Pull request by towerofnix**
Fixes #19. :gnu_slash_linux:

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    <program>  Copyright (C) <year>  <name of author>
+    This program comes with ABSOLUTELY NO WARRANTY; for details type `show w'.
+    This is free software, and you are welcome to redistribute it
+    under certain conditions; type `show c' for details.
```
Comment (heyitsmeuralex) [inline LICENSE.txt]: Implement this thanks

Comment (towerofnix) [reply LICENSE.txt]: Done, more or less - use the `license` command. (Also added an intro statement, which directs people to using `license` and `help`.)

**Comment by towerofnix** [comment]
@heyitsmeuralex how so? I don't think there's anything in the GPL that requires a README document? Although noting the GPL within the README is definitely worth doing. (I'd say this is the other way around - readme is blocked by this PR :) )

**Comment by heyitsmeuralex** [comment]
ok maybe not
<h6>you said in a #26 that this was blocked so i got confused</h6>

**Comment by towerofnix** [comment]
No, I didn't - I linked to #22 ("fill in package.json") and its related PR #24, not this PR #23. :tada:

**Pull request merged**